modules/auxiliary/server/capture/smb.rb

```diff
@@ -193,8 +193,9 @@ def on_cred(creds)
 
   def run
     @rsock = Rex::Socket::Tcp.create(
-      'LocalHost' => datastore['SRVHOST'],
-      'LocalPort' => datastore['SRVPORT'],
+      'LocalHost' => bindhost,
```

Comment:
I think this might be an existing issue, but are there any limitations with this solution in the scenario of a module wanting to have multiple different services with different bindports? For instance wanting to run an http server as well as an smb server in a module? 🤔

Reply:
Comment not specific to this line of code, just an open question 😄